### PR TITLE
Model distribute parameter

### PR DESCRIPTION
**Pull request by zaucker**
First step if issue #440 
- Rename `branch` and `adjustBranching` to `distribute` in model and test models
- Adapt backend

### CODE REVIEW

File: lib/Agrammon/Model/Input.pm6 [hunk 1]
```diff
@@ -20,19 +20,19 @@ class Agrammon::Model::Input {
...
         }
         with $default_gui {
             $!default-gui = val($_);
         }
-        with $branch {
+        with $distribute {
             if .lc eq 'true' {
-                $!branch = True;
+                $!distribute = True;
             }
```
Comment (jnthn) [inline lib/Agrammon/Model/Input.pm6:36]: Coulda just been `$!distribute = .lc eq 'true';`

**Review by jnthn — APPROVED** [review]
A tiny cleanup is possible, I'll take it on during my work on the next steps for multiple branches.

File: lib/Agrammon/Model/Input.pm6 (hunk 1 above)
Comment (zaucker) [reply lib/Agrammon/Model/Input.pm6:36]: Fixed. Will push after rebase.

Comment (jnthn) [reply lib/Agrammon/Model/Input.pm6:36]: Oops, noticed this after merging. Will do it in my branch.

**Pull request merged**